public/index.html

```diff
@@ -0,0 +1,370 @@
+<!doctype html>
+<html class="no-js" lang="en">
+  <head>
+    <meta charset="utf-8">
+    <meta http-equiv="x-ua-compatible" content="ie=edge">
+    <title>Avengers</title>
+    <meta name="viewport" content="width=device-width, user-scalable=no">
+    <link rel="apple-touch-icon" href="apple-touch-icon.png">
+    <link rel="stylesheet" href="css/reset.css">
+    <link rel="stylesheet" href="css/icon-styles.css">
+    <link rel="stylesheet" href="css/main.css">
+  </head>
+  <body>
+    <!-- menu -->
+    <nav class="nav top-nav">
+      <ul class="flex-container flex-navbar">
+        <li class="flex-nav-1"><a href="#home">Home.</a></li>
+        <li class="flex-nav-2"><a href="#">Services.</a></li>
+        <li class="flex-nav-3"><a href="#">Portfolio.</a></li>
+        <li class="flex-nav-4"><a href="#">About.</a></li>
+        <li class="flex-nav-5"><a href="#">Team.</a></li>
+        <li class="flex-nav-6"><a href="#">Contact.</a></li>
+      </ul>
+    </nav>
+    <!-- header -->
+    <header class="flex-container header-notch">
+      <section class="headertext">
+        <h1>
+          <span class="icon-droplet"></span><br>
+          Hi there! We're Avengers, <strong>creative design agency</strong>.
+          We build <span class="underline"><strong>awesome</strong></span> mobile and websites.
+        </h1>
+      </section>
+    </header>
+    <!-- services  -->
+    <article class="services">
+      <h1>SERVICES</h1>
+      <section class="flex-container flex-paragraphs">
+        <p class="flex-paragraph-1">
+          Contrary to popular belief, Lorem Ipsum is not simply random text. It has roots in a piece of classical Latin literature from 45 BC, making it over 2000 years old. Richard McClintock, a Latin professor at Hampden-Sydney College in Virginia.
+        </p>
+        <p class="flex-paragraph-2">
+          Lorem Ipsum is simply dummy text of the printiand typesetting industry. Lorem Ipsum has been the industry's standadummy text ever since the 1500s, when an unknown printer took a gallof type and scrambled it to make a type specimen book.
+        </p>
+        <p class="flex-paragraph-3">
+          Lorem Ipsum is simply dummy text of the printiand typesetting industry. Lorem Ipsum has been the industry's standadummy text ever since the 1500s, when an unknown printer took a gallof type and scrambled it to make a type specimen book.
+        </p>
+      </section>
+      <section class="selling-points">
+        <section class="strengths">
```

Comment:
`section` inception. 

According to the spec: 

```
A section, in this context, is a thematic grouping of content. Each section should be identified, typically by including a heading ( h1- h6 element) as a child of the section element.
```

So at the very least, the outer `section` should contain an `h1-h6`. But I don't think a `section` is appropriate for either element in this case.